openSUSE/open-build-service
# Use component to delete group members

Use DeleteConfirmationDialogComponent instead of a partial to render the confirmation dialog.

To test this

- log in as Admin
- go to Configuration > Manage Groups
- click on the name of one of the groups, you'll see its members
- remove one member (you can add some first)
- check if  he name of the member is in the confirmation dialog
- click on "Remove" and check if the member disappears from the group
